dot-net-sdk/helpers/AppDetails.cs

```diff
@@ -44,7 +46,8 @@ public string GetVersion()
         return new Dictionary<string, string>() {
             ["sdkLanguage"] = SDK_LANG,
             ["sdkName"] = GetName(),
-            ["sdkVersion"] = GetVersion()
+            ["sdkVersion"] = GetVersion(),
+            ["clientUID"] = _uuid
```

Comment:
Thanks for tackling this - really looking forward to adding the new functionality. I was doing some brainstorming in Slack and needed to work out an important detail:

We use Fastly as the CDN and the query parameters make up the cache key - it would negatively impact the cache hit rate to include a uuid in that payload.

Luckily Fastly does not cache headers - we have the request logs hooked up to append to a BigQuery table. Whether its a cache hit or a miss a row is added.

I submitted a PR to add `X-Eppo-Client-Identifier` - since the US team is OOO I will get back to you Monday after it is accepted.

Reply:
Great idea to use a header.